### PR TITLE
Solve the issue of console flicker

### PR DESCRIPTION
**Pull request by Jim8y**
Maybe not a big deal for those who do not check the synchronization states a lot, however, as a NEO community developer, I use `show state` command a lot.  Though it works fine in my Nvidia powered monitor, it flickers in another one every time the state updates. Apart from that, in both of my monitors, if I click the console while `show state` is executing, the whole console turn black.

### CODE REVIEW

File: neo-cli/Shell/MainService.cs [hunk 1]
```diff
@@ -818,24 +818,38 @@ private bool OnShowPoolCommand(string[] args)
...
                 {
                     uint wh = 0;
+                    Console.CursorTop = 0;
+                    Console.CursorLeft = 0;
                     if (Program.Wallet != null)
                         wh = (Program.Wallet.WalletHeight > 0) ? Program.Wallet.WalletHeight - 1 : 0;
-                    Console.Clear();
-                    Console.WriteLine($"block: {wh}/{Blockchain.Singleton.Height}/{Blockchain.Singleton.HeaderHeight}  connected: {LocalNode.Singleton.ConnectedCount}  unconnected: {LocalNode.Singleton.UnconnectedCount}");
+                    var output = $"block: {wh}/{Blockchain.Singleton.Height}/{Blockchain.Singleton.HeaderHeight}  connected: {LocalNode.Singleton.ConnectedCount}  unconnected: {LocalNode.Singleton.UnconnectedCount}";
+                    Console.Write(output.PadRight(output.Length));
```
Comment (erikzhang) [inline neo-cli/Shell/MainService.cs]: What is `PadRight()` used for?

Comment (Liaojinghui) [reply neo-cli/Shell/MainService.cs]: Sorry, PadRight is unnecessary, removed in a new commit.

File: neo-cli/Shell/MainService.cs [hunk 2]
```diff
@@ -818,24 +818,36 @@ private bool OnShowPoolCommand(string[] args)
...
-                    Console.Clear();
-                    Console.WriteLine($"block: {wh}/{Blockchain.Singleton.Height}/{Blockchain.Singleton.HeaderHeight}  connected: {LocalNode.Singleton.ConnectedCount}  unconnected: {LocalNode.Singleton.UnconnectedCount}");
+                    Console.Write($"block: {wh}/{Blockchain.Singleton.Height}/{Blockchain.Singleton.HeaderHeight}  connected: {LocalNode.Singleton.ConnectedCount}  unconnected: {LocalNode.Singleton.UnconnectedCount}");
+                    consoleLine = 1;
                     foreach (RemoteNode node in LocalNode.Singleton.GetRemoteNodes().Take(Console.WindowHeight - 2))
-                        Console.WriteLine($"  ip: {node.Remote.Address}\tport: {node.Remote.Port}\tlisten: {node.ListenerPort}\theight: {node.Version?.StartHeight}");
+                    {
+                        Console.CursorTop = consoleLine++;
+                        Console.CursorLeft = 0;
+                        Console.Write($"  ip: {node.Remote.Address}\tport: {node.Remote.Port}\tlisten: {node.ListenerPort}\theight: {node.Version?.StartHeight}");
```
[the remark this answers is not in the document]
Comment (Liaojinghui) [reply neo-cli/Shell/MainService.cs]: Hello Erik, thank you for reviewing my code. And I am sorry that I didn't consider the condition properly. As to the length of the string, I added a `\t` at the end of the string to make sure the length will always be the same. And I also added extra code to handle the situation where the number of connections drops.

**Comment by erikzhang** [comment]
@Liaojinghui Hi, I've made some changes, please review it.

**Pull request merged**